docker-compose.yml

```diff
@@ -45,8 +46,7 @@ services:
     ports:
       - 8443:8443
     depends_on:
-      fineractmysql:
-        condition: service_healthy
+      - fineractmysql
```

Comment:
These two expressions are not doing the same thing. "condition: service_healthy" is actually intended an more accurate than what you suggest here.